server/services/prismic-content.js

```diff
@@ -44,17 +44,33 @@ export function getFeaturedMediaFromBody(doc): ?Picture {
 
 export function prismicImageToPicture(captionedImage) {
   const image = isEmptyObj(captionedImage.image) ? null : captionedImage.image;
+  const attribute = image && image.copyright && getTaslFromCopyright(image.copyright);
+
   return ({
     type: 'picture',
     contentUrl: image && convertPrismicToImgIxUri(image.url), // TODO: Send this through the img.wc.org
     width: image && image.dimensions.width,
     height: image && image.dimensions.height,
     caption: captionedImage.caption && captionedImage.caption.length !== 0 && asHtml(captionedImage.caption),
     alt: image && image.alt,
-    copyrightHolder: image && image.copyright
+    copyrightHolder: image && image.copyright,
+    attribute
   }: Picture);
 }
 
+function getTaslFromCopyright(copyright) {
+  // We expect a string of title|author|source|license
+  // e.g. |Rob Bidder||CC-BY-NC
+  const list = copyright.split('|');
+  const v = list
+    .concat(Array(4 - list.length))
+    .map(v => !v.trim() ? null : v);
+
+  const [title, author, source, license] = v;
+
+  return {title, author, source, license};
```

Comment:
Could we chuck copyrightHolder and copyrightHolderLink in here too and drop it above?

So we might have something like:
`Ocean views | Greg M | https://wc.org/things/2837 | CC-BY-NC | Wellcome Images | https://wi.org`

Reply:
Yep, good idea.